contracts/app/Bridge.sol

```diff
@@ -99,18 +107,30 @@ contract BridgeDestination {
         view
         returns (uint256)
     {
-        // TODO:
-        return 0;
+        uint256 currentTime = block.timestamp;
+        if (currentTime < startTime){
+            return 0;
+        }else if(currentTime >= transferData.startTime + transferData.feeRampup){
+            return transferData.fee;
+        }else{
+            return transferData.fee * (transferData.currentTime - transferData.startTime); 
```

Comment:
Should be currentTime (instead of transferData.currentTime?). Further, should this value exceed the fee?